plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/CloudLibrariesPage.java

```diff
@@ -87,6 +95,13 @@ public CloudLibrariesPage() {
   public void initialize(IJavaProject javaProject, IClasspathEntry[] currentEntries) {
     this.project = javaProject;
     isMavenProject = MavenUtils.hasMavenNature(javaProject.getProject());
+    existingEntries = currentEntries != null ? currentEntries : new IClasspathEntry[0];
```

Comment:
`existingEntries` doesn't need to be a field. That said, I think this is more readable instead of always using `Stream.of()`. 

`originalEntry = currentEntries == null ? null : Stream.of(currentEntries)....`